doc/classes/CPUParticles2D.xml

```diff
@@ -68,132 +93,194 @@
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set orbital_velocity properties.
 		</constant>
 		<constant name="PARAM_LINEAR_ACCEL" value="3" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set linear acceleration properties.
 		</constant>
 		<constant name="PARAM_RADIAL_ACCEL" value="4" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set radial acceleration properties.
 		</constant>
 		<constant name="PARAM_TANGENTIAL_ACCEL" value="5" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set tangential acceleration properties.
 		</constant>
 		<constant name="PARAM_DAMPING" value="6" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set damping properties.
 		</constant>
 		<constant name="PARAM_ANGLE" value="7" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set angle properties.
 		</constant>
 		<constant name="PARAM_SCALE" value="8" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set scale properties.
 		</constant>
 		<constant name="PARAM_HUE_VARIATION" value="9" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set hue_variation properties.
 		</constant>
 		<constant name="PARAM_ANIM_SPEED" value="10" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set animation speed properties.
 		</constant>
 		<constant name="PARAM_ANIM_OFFSET" value="11" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set animation offset properties.
 		</constant>
 		<constant name="PARAM_MAX" value="12" enum="Parameter">
+			Represents the size of the [enum Parameter] enum.
 		</constant>
 		<constant name="FLAG_ALIGN_Y_TO_VELOCITY" value="0" enum="Flags">
+			Use with [method set_flag] to set [member flag_align_y].
 		</constant>
 		<constant name="FLAG_ROTATE_Y" value="1" enum="Flags">
+			Use with [method set_flag] to set [member flag_rotate_y].
 		</constant>
 		<constant name="FLAG_DISABLE_Z" value="2" enum="Flags">
+			Use with [method set_flag] to set [member flag_disable_z].
 		</constant>
 		<constant name="FLAG_MAX" value="3" enum="Flags">
+			Represents the size of the [enum Flags] enum.
 		</constant>
 		<constant name="EMISSION_SHAPE_POINT" value="0" enum="EmissionShape">
+			All particles will be emitted from a single point.
 		</constant>
 		<constant name="EMISSION_SHAPE_CIRCLE" value="1" enum="EmissionShape">
+			Particles will be emitted in the area of a circle.
```

Comment:
Currently they will be emitted on the *outline* of a circle. That's a inconsistency with `Particles2D` noted in #29692.